playbooks/special_collections.yml

```diff
@@ -10,17 +13,17 @@
         - run a cap for special_collections
         - where needed make sure server is greenlisted on [lib-ponyexpress](https://github.com/pulibrary/pul-the-hard-way/blob/master/services/smtp-mail-server.md)
   vars_files:
-    - ../group_vars/special_collections/special_collections_prod.yml
+    - ../group_vars/special_collections/{{ runtime_env | default('staging') }}.yml
     - ../group_vars/special_collections/vault.yml
   roles:
     - role: datadog
```

Comment:
Only run the datadog role on production servers.

Reply:
the CI breaks without ezproxy in it. So I cherry picked it from main.